docs/spelling_wordlist

```diff
@@ -790,7 +790,7 @@ versioning
 vertices
 viewable
 virtualized
-Weblog
+weblog
```

Comment:
I think we can change `weblog` to `blog` as a part of these changes and remove it from the word-list.